app/presenters/link_api_response_presenter.rb

```diff
@@ -19,7 +19,7 @@ def local_authority_details
       "local_authority" => {
         "name" => @authority.name,
         "snac" => @authority.snac,
-        "tier" => @authority.tier,
+        "tier" => Tier.as_string(@authority.tier_id),
```

Comment:
I know this has been merged already, but quick thought. Would it be more convenient to add a `tier` method on the LocalAuthority model that calls this method. That might make the LocalAuthority model a bit more conventional in terms of how it deals with tiers...?